manifests/r/Resolume/Avenue/7.21.0.37521/Resolume.Avenue.locale.en-US.yaml

```diff
@@ -0,0 +1,31 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Resolume.Avenue
+PackageVersion: 7.21.0.37521
+PackageLocale: en-US
+Publisher: Resolume
+PublisherUrl: https://resolume.com
+PublisherSupportUrl: https://resolume.com/support/
+PrivacyUrl: https://resolume.com/company/privacy

```

Comment:
```suggestion
# PrivacyUrl:
```

Reply:
Fixed privacy policy was moved with our new website but no auto redirect rule in place.